hubs.yaml

```diff
@@ -38,21 +38,15 @@ clusters:
                 funded_by:
                   name: 2i2c
                   url: https://2i2c.org
-            auth:
-              # Admin names need to be listed twice here, unfortunately
-              # 'whitelist' will be removed by next jupyterhub release
-              whitelist:
-                users:
-                  - yuvipanda@gmail.com
-                  - colliand@gmail.com
-                  - choldgraf@gmail.com
-                  - georgiana.dolocan@gmail.com
-              admin:
-                users:
-                  - yuvipanda@gmail.com
-                  - colliand@gmail.com
-                  - choldgraf@gmail.com
-                  - georgiana.dolocan@gmail.com
+            hub:
+              config:
+                Authenticator:
+                  allowed_users: &staging_users
+                    - yuvipanda@gmail.com
+                    - colliand@gmail.com
+                    - choldgraf@gmail.com
+                    - georgiana.dolocan@gmail.com
+                  admin_users: *staging_users
```

Comment:
Users added to admin_users doesn't need to also be added to allowed_users, right? No damage done doing this though.